apps/dashboard/pkg/migration/conversion/v0.go

```diff
@@ -25,8 +25,25 @@ func Convert_V0_to_V1(in *dashv0.Dashboard, out *dashv1.Dashboard, scope convers
 	if err := migration.Migrate(out.Spec.Object, schemaversion.LATEST_VERSION); err != nil {
 		out.Status.Conversion.Failed = true
 		out.Status.Conversion.Error = err.Error()
+
+		logger.Error("Dashboard conversion failed",
+			"sourceVersionAPI", dashv0.APIVERSION,
+			"targetVersionAPI", dashv1.APIVERSION,
+			"dashboardUID", in.ObjectMeta.UID,
+			"sourceSchemaVersion", in.Spec.Object["schemaVersion"],
+			"targetSchemaVersion", schemaversion.LATEST_VERSION,
+			"error", err)
+
```

Comment:
May also then be helpful here to have a total count of failures via a metric too :)

Reply:
Sure!!! I'm doing that! Should we keep the logging error? Or is the error something we can capture in another way?

Reply:
Keeping the error logged is great :) thanks for adding the metrics!!